pkg/common/v1alpha1/common.go

```diff
@@ -1,4 +1,4 @@
-package pkg
+package v1alpha1
```

Comment:
Maybe name this package `common` ?

Reply:
based on go best practices, package name should keep same as its directory name. all Katit package follows this style.

Reply:
Yeah, I think we should keep v1alpha1